frontend/packages/dev-console/integration-tests/features/guided-tour/creating_a _serverless_application.feature

```diff
@@ -0,0 +1,268 @@
+Feature: Creating a Serverless application tour
+	As a user, I want to take a guided tour of creating a serverless application feature   
+
+Background:
+    Given user is in developer perspective
+
+@regression
+Scenario: Starting tour from the +Add page 
+   Given user is in +Add page 
+   And user sees quick tour card 
+   When user clicks on the "Creating a Serverless application" link on the card
+   And user sees the tour will start as a sidescreen with close button
+   And user sees the link to five steps present in the card
+   And user clicks on the Start tour option
+   And user sees "Create a Serverless application" step is started
+   And user sees steps to create an Serverless application
+   And user clicks on next
+   And user sees an alert appears "Check your work" asking to verify that the Serverless Operator was successfully installed 
+   And user selects Yes option
+   And user clicks on next
+   And user sees next step "Demo scalability" started
+   And user sees steps to see your application scale
+   And user clicks on next
+   And user sees an alert appears "Check your work" asking to verify that your application scaled successfully 
+   And user selects Yes option
+   And user clicks on next
+   And user sees next step "Wire an event source to your Knative Service" started 
+   And user sees steps to wire an event source to your Knative Service
+   And user clicks on next
+   And user sees an alert appears "Check your work" asking to wire an event source to your Knative Service successfully 
+   And user selects Yes option
+   And user clicks on next
+   And user sees next step "Force a new revision & set traffic distribution" started
+   And user sees steps to force a new revision & set traffic distribution
+   And user clicks on next
+   And user sees an alert appears "Check your work" asking to verify that you forced a new revision & set traffic distribution successfully 
+   And user selects Yes option
+   And user clicks on next
+   And user sees next step "Delete your application" started
+   And user sees steps to delete your application you just created
+   And user clicks on next
+   And user sees an alert appears "Check your work" asking to verify that your application scaled successfully 
+   And user selects Yes option
+   And user clicks on next
+   Then user sees the security alert appearing on top saying "This tour has already been completed"
+   And user sees Knative Cookbook link
+   And user sees "Close", "Back" and "View all tours"
+   And user clicks "View all tours" button to see all tour options in background
```

Comment:
Then statement should not perform any action. Can you please update it?

Reply:
Those are required actions that need to be present in the given result